tests/migration_test.go

```diff
@@ -4128,3 +4128,29 @@ func disableNodeLabeller(node *k8sv1.Node, virtClient kubecli.KubevirtClient) *k
 
 	return node
 }
+
+func LibvirtDomainIsPersistent(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (bool, error) {
+	vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
```

Comment:
Just a note, we had here:
```
	vmiPod, err := getRunningPodByVirtualMachineInstance(vmi, util2.NamespaceTestDefault)
	if err != nil {
		return false, err
	}
```
Now we have `util2.PanicOnError(err)` [there](https://github.com/benukhanov/kubevirt/blob/19e78a2ec267db569acf8c1e662a52332df8d6a1/tests/utils.go#L845).

Reply:
@codingben Hi. Thank you for your PR! you can add such notes in the PR description under the **special notes for the reviewer**.

Reply:
Yep, sounds good. Thanks!